pkg/kafka/auth/scram.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"crypto/sha256"
+	"crypto/sha512"
+	"hash"
+
+	"github.com/Shopify/sarama"
+	"github.com/pkg/errors"
+	SCRAM "github.com/xdg/scram"
+)
+
+// scramConfig describes the configuration properties required for the SCRAM handshake
+// between the collector and the pipeline
+type scramConfig struct {
+	userName  string `mapstructure:"userName"`
```

Comment:
username is usually written a single word, lowercase

Reply:
Changed username for map structure.